Makefile.am

```
@@ -102,78 +99,62 @@ AM_CXXFLAGS = $(OPENMP_CXXFLAGS)
 
 # Rules for src/api.
 
-libtesseract_api_la_CPPFLAGS = $(AM_CPPFLAGS)
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/arch
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/ccmain
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/ccstruct
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/ccutil
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/classify
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/cutil
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/dict
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/opencl
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/textord
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/viewer
-libtesseract_api_la_CPPFLAGS += -I$(top_srcdir)/src/wordrec
+libtesseract_la_CPPFLAGS = $(AM_CPPFLAGS)
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/arch
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/ccmain
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/ccstruct
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/ccutil
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/classify
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/cutil
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/dict
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/lstm
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/opencl
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/textord
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/training
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/viewer
+libtesseract_la_CPPFLAGS += -I$(top_srcdir)/src/wordrec
 libtesseract_api_la_CPPFLAGS += $(libcurl_CFLAGS)
```

This line is now stranded. This is the cause of the LGTM failure:
```
Makefile.am:116: error: libtesseract_api_la_CPPFLAGS must be set with '=' before using '+='
```
```suggestion
libtesseract_la_CPPFLAGS += $(libcurl_CFLAGS)
```

---

That was a bad merge conflict resolution. It is fixed now.